Export.FCS.sln

```diff
@@ -0,0 +1,27 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
It's probably best if all files that have something to do with Fable have Fable in the name where possible?